packages/flame/lib/src/components/core/component.dart

```diff
@@ -855,12 +855,11 @@ class Component {
   int _priority;
   set priority(int newPriority) {
     if (_priority != newPriority) {
+      _priority = newPriority;
       final parent = _parent;
       final game = findGame();
       if (game != null && parent != null) {
         game.enqueuePriorityChange(parent, this, newPriority);
```

Comment:
we don't need to pipe in the newPriority anymore right?

Reply:
What do you mean?

Reply:
Aaah now I get what you mean, it showed the comment on the wrong line on the phone. 😅

Reply:
Fixed in [9233b40](https://github.com/flame-engine/flame/pull/3564/commits/9233b408306c5b447311f2fb3248140f000e51f6)